src/RendererDX11.cpp

```diff
@@ -416,25 +409,10 @@ void RendererDX11::set_light_buffer(std::shared_ptr<Drawable> const& drawable) c
         light_data.directional_light.ambient = m_directional_light->ambient;
         light_data.directional_light.diffuse = m_directional_light->diffuse;
         light_data.directional_light.specular = m_directional_light->specular;
+        light_data.directional_light.light_projection_view = m_directional_light->get_projection_view_matrix();
     }
 
-    i32 num_point_lights;
```

Comment:
That whole change with the number of lights, removing sorting of the lights, etc. should be a commit of its own

Reply:
Yup, I know, will split that.